webpack.config.js

```diff
@@ -72,6 +73,7 @@ module.exports = {
         extensions: [".js", ".jsx"]
     },
     devServer: {
-        port: 3001
+        port: 3000,
+        historyApiFallback: true
```

Comment:
this is the webpack config, as you saw in `server.js`, webpack runs as an express middleware - when the server is started, webpack bundles up all of the client side javascript